pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -555,6 +580,15 @@ func (cr *PerconaXtraDBCluster) CheckNSetDefaults(serverVersion *version.ServerV
 		}
 	}
 
+	if c.PXC.Expose.Enabled {
+		if c.PXC.Expose.TrafficPolicy == "" {
```

Comment:
We can't use both `Type: ClusterIP` and `ExternalTrafficPolicy: Local` as defaults: `spec.externalTrafficPolicy: Invalid value: "Local": ExternalTrafficPolicy can only be set on NodePort and LoadBalancer service`

Reply:
Good catch!
fixed 15b53dfd